crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -32,7 +32,8 @@ int crypto_sign_keypair_internal(ml_dsa_params *params,
   uint8_t tr[TRBYTES];
   const uint8_t *rho, *rhoprime, *key;
   polyvecl mat[DILITHIUM_K_MAX];
-  polyvecl s1, s1hat;
+  polyvecl s1 = {{0}};
```

Comment:
Oh, `polyvecl` is a struct which contains an array of `poly`, which is an array of `int32 coeffs`
```suggestion
  polyvecl s1 = {{{0}}};
```